avocado/plugins/spawners/podman.py

```diff
@@ -79,6 +81,8 @@ class PodmanSpawner(DeploymentSpawner, SpawnerMixin):
     description = 'Podman (container) based spawner'
     METHODS = [SpawnMethod.STANDALONE_EXECUTABLE]
 
+    _PYTHON_VERSIONS_CACHE = {}
```

Comment:
Eventually we want this cache to be persisted across `avocado` executions, but this is fine for now.